oak_functions/loader/src/main.rs

```diff
@@ -190,12 +215,18 @@ async fn main() -> anyhow::Result<()> {
     let wasm_module_bytes = fs::read(&opt.wasm_path)
         .with_context(|| format!("Couldn't read Wasm file {}", &opt.wasm_path))?;
 
-    let lookup_data = LookupData::new_empty(&config.lookup_data_url);
+    let lookup_data = Arc::new(LookupData::new_empty(&config.lookup_data_url));
     if !config.lookup_data_url.is_empty() {
-        lookup_data
-            .refresh()
-            .await
-            .context("could not load lookup data")?;
+        let lookup_data_download_period = match config.lookup_data_download_period {
+            Some(v) => v,
+            None => {
```

Comment:
Should we enforce this? I can imagine cases where the data is stable enough that refreshes might not be required outside of the normal restarts of the runtime.

Reply:
Good point, made more lenient now.